### PR TITLE
Prismic recursive rendering

### PR DESCRIPTION
**Pull request by defudef**

### CODE REVIEW

**Review by filrak — CHANGES_REQUESTED** [review]
Prismic is not a part of commercetools so it shouldnt be a part of commercetools integration

File: packages/commercetools/theme/components/prismic/CustomPage.vue [hunk 1]
```diff
@@ -0,0 +1,81 @@
+<template>
+  <div>
+    {{blocks.length}}
```
Comment (filrak) [inline packages/commercetools/theme/components/prismic/CustomPage.vue]: ^^

Comment (defudef) [inline packages/commercetools/theme/components/prismic/CustomPage.vue]:
```suggestion
```

File: packages/commercetools/theme/package.json [hunk 2]
```diff
@@ -12,7 +12,8 @@
     "test": "jest"
   },
   "dependencies": {
-    "@storefront-ui/vue": "^0.5.1",
+    "@nuxtjs/prismic": "^1.0.0",
+    "@storefront-ui/vue": "file:../../../../storefront-ui/packages/vue",
```
Comment (filrak) [inline packages/commercetools/theme/package.json]: Shoupd be 0.5.1

**Review by filrak — CHANGES_REQUESTED** [review]

File: packages/commercetools/theme/package.json [hunk 3]
```diff
@@ -2,6 +2,7 @@
   "name": "theme",
   "description": "My breathtaking Nuxt.js project",
   "author": "Filip Rakowski",
+  "version": "1.0.1",
```
Comment (defudef) [inline packages/commercetools/theme/package.json]:
```suggestion
  "version": "0.0.3",
```

**Review by filrak — APPROVED** [review]

**Pull request merged**